configure.in

```diff
@@ -7263,6 +7260,10 @@ if test "$USE_WIN32" = 1 ; then
         fi
     fi
 
+    if test "$wxUSE_DIB" = "yes"; then
+        AC_DEFINE(wxUSE_WXDIB)
```

Comment:
I have decided to add this feature after finishing this pull commits.
From http://trac.wxwidgets.org/ticket/13902#comment:32
kosenko wrote "Enabled some Windows features: DIB ..."

The work is big enough for a separate pull.

Tim S.